amida-tech/indaba-client
# Bug/INBA-527 Fix new messages from icons

#### What does this PR do?
Fixes the mail icons in the users list in the project manager and the all users list

#### Related JIRA tickets:
[INBA-517](https://jira.amida-tech.com/browse/INBA-517)

#### How should this be manually tested?
1. Click on a mail link in the users list in the project manager and check that a new message interface is spawned with that user in the to: field
1. Click on a mail link in the All users list and check that a new message interface is spawned with that user in the to: field

#### Background/Context

#### Screenshots (if appropriate):
